src/index.html

```diff
@@ -11,7 +11,180 @@
+            name="terms"
+            class="terms"
+            id="terms"
+            value="agree"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label for="email" class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label for="password" class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            class="password"
+            minlength="6"
+            maxlength="18"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label for="loveCatsYes" class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsYes"
+            class="radio"
+            value="yes"
+          >Yes
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsNo"
+            class="radio"
+            value="no"
+          >No
+        </label>
+        <label for="color" class="label">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favColor"
+            id="color"
+            class="color"
+          >
+        </label>
+        <label for="bed-time" class="label">
+          What time do you go to bed?
+          <input
+            type="time"
+            step="2"
+            name="bedTime"
+            id="bed-time"
+            class="bed-time"
+          >
+        </label>
+        <label for="fav-car" class="label">
+          What are your favorite brand of cars?
+          <select
+            name="favCar"
+            id="fav-car"
+            class="fav-car-list"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label for="rating" class="label">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rating"
+            value="0"
+            id="rating"
+            class="rating"
+            min="0"
+            max="10"
+            step="1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+        <label for="comments" class="label">
+          Comments:
+          <textarea
+            name="comments"
+            id="comments"
+            class="comments"
+            cols="20"
+            rows="2"
+          ></textarea>
+        </label>
+        <label for="name" class="label">
```

Comment:
for attribute seem redundant here
```suggestion
        <label class="label">
```